### DISCUSSION TITLE
questiong about the unit system in MOOSE

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#28057] [Q&A Modules: General]
I am running an inelastic numerical simulation in which I set the loading boundary condition to 0.005*t, but at t=1000s, the average elemental displacement is 5000 instead of 5mm, so I am a little confused about how the unit system works in MOOSE

**Comment by GiudGiud** [comment]
Hello

For the others, the units in MOOSE are first determined by the units of the mesh (for length), then the input has to be consistent with those.
